spec/factories/other_assets_declarations.rb

```diff
@@ -0,0 +1,25 @@
+FactoryBot.define do
+  factory :other_assets_declaration do
+    initialize_with { create(:legal_aid_application).other_assets_declaration }
+
+    trait :with_second_home do
+      second_home_value { 850_000.00 }
```

Comment:
I'd recommend using a random number generator in factories (native ruby or via Faker for example), as that avoids the possibility that the specific number specified in the factory is some sort of magic number that means in allows specs to pass when other numbers would cause an error.